plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -6,6 +6,39 @@ const webpack = require('webpack');
 const config = require('../config/webpack.config');
 const args = require('minimist')(process.argv.slice(2))
 
+const fs = require('fs')
+const FileManagerPlugin = require('filemanager-webpack-plugin');
+const envPaths = require('env-paths');
+
+
+/**
+ * Copies the built plugin to the app config folder ~/.config/Headlamp/plugins/
+ */
+function copyToPluginsFolder(config) {
+  const packageJson = JSON.parse(fs.readFileSync('package.json', 'utf8'))
+
+  // @todo: should the whole package name be used here, 
+  //    and the load be fixed to use? What about namespace packages?
+  const packageName = packageJson.name.split('/').splice(-1)[0];
+  const paths = envPaths('Headlamp', {suffix: ''});
+
+  config.plugins = [
+    new FileManagerPlugin({
+      events: {
+        onEnd: {
+          copy: [{
+            source: './dist/*',
+            destination: `${paths.config}/plugins/${packageName}/`,
+          }],
+        },
+      }
+    }),
+  ];
+}
```

Comment:
Would it work if we just used a symlink? (not sure if there are implications on Windows)
The advantage would be that it'd be faster, and if we also name it with a known prefix (e.g. `__dev_`+plugin_name) in order to clean up broken links (once development is finished).

Reply:
I'm pretty sure you need some special permissions on windows for symlinks. I had to do this recently, and required to run as admin.